src/tokens/AssetList.tsx

```diff
@@ -277,6 +279,7 @@ export default function AssetList({
   return (
     <AnimatedSectionList
       contentContainerStyle={[
+        { minHeight: variables.height + (isWalletTab ? 0 : headerHeight) },
```

Comment:
guess its only short term, but will be good to document why headerHeight is needed when on drawer view

Reply:
Added documentation in 2b339cc03064238a701a1443f670e22d24b5308c and bda5c30c783b41244b30a8b86e462ff0311bda79 (typo-fix)!